lib/releaselib.sh

```diff
@@ -927,8 +929,8 @@ release::docker::release () {
 
     # 'gcloud docker' gives lots of internal_failure's so add retries to
     # all of the invocations
-    for arch in "${KUBE_SERVER_PLATFORMS[@]##*/}"; do
-      for binary in "${binaries[@]}"; do
+    for binary in "${binaries[@]}"; do
```

Comment:
Why reverse the loop flow here?

Reply:
This reverse will help me running manifest in outer loop for every binary.

Reply:
This branch of the if/else is defunct starting with kubernetes 1.8 (https://github.com/kubernetes/kubernetes/pull/47939). You should probably add this new logic to the `release::docker::release_from_tarfiles` function instead.